### PR TITLE
[RW-4737][risk=no] Enable Google Private IP Adddress flag on new project creation

### PR DESCRIPTION
**Pull request by jaycarlton**
Incorporate new RAWLS option into our version of the FireCloud Swagger, and set the new option to true.

I did see a couple of attempts fail before this one succeeded, so I'm not sure what happened. After restarting everything, it worked.

Screen Shot 2020-04-18 at 2 00 20 PM
---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/interceptors/AuthInterceptor.java [hunk 1]
```diff
@@ -171,7 +171,7 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
             new UserAuthentication(user, OAuth2Userinfo, token, UserType.RESEARCHER));
 
     // TODO: setup this in the context, get rid of log statement
-    log.log(Level.INFO, "{0} logged in", OAuth2Userinfo.getEmail());
+    log.log(Level.FINE, "{0} logged in", OAuth2Userinfo.getEmail());
```
Comment (calbach) [inline api/src/main/java/org/pmiops/workbench/interceptors/AuthInterceptor.java:174]: This was a very critical log line for debugging in GAE logs. The change is also entirely unrelated to the PR. Please revert this.

Comment (jaycarlton) [reply api/src/main/java/org/pmiops/workbench/interceptors/AuthInterceptor.java:174]: OK. It's an annoyance locally and doesn't give me much context about what was going on. I thought it was just leftover and tried to clean it up.

Comment (jaycarlton) [reply api/src/main/java/org/pmiops/workbench/interceptors/AuthInterceptor.java:174]: Is it supposed to fire every minute?

Comment (calbach) [reply api/src/main/java/org/pmiops/workbench/interceptors/AuthInterceptor.java:174]: No, though it might if you have local UI  tab open that's polling your API server. If you paste the logs you observed, I can try and answer. The log line is expected to run on every authenticated request.
